rock_paper_scissor.py

```diff
@@ -0,0 +1,106 @@
+import random
+
+
+class RpsGame:
+    def __init__(self, max_attempts):
+        self._current_attempt = 0
+        self._max_attempts = max_attempts
+        self._comp_choice = None
+        self.user_score = 0
+        self.comp_score = 0
+
+    def increment_user_score(self):
+        self.user_score += 1
+
+    def increment_comp_score(self):
```

Comment:
Can we make this method as private, since it is only used from within the class method?